website/Gemfile.lock

```diff
@@ -79,18 +79,18 @@ GEM
       celluloid (~> 0.16.0)
       rb-fsevent (>= 0.9.3)
       rb-inotify (>= 0.9)
-    middleman (3.3.13)
+    middleman (3.3.12)
```

Comment:
I just moved this change into a separate PR - #2073 to make it easier for anyone to review.